svg/elements/feComposite.json

```diff
@@ -202,28 +226,28 @@
                 "version_added": "≤79"
               },
               "firefox": {
-                "version_added": null
+                "version_added": true
```

Comment:
according to Robert `in`, `in2` and all the `k` values are supported by everything. So these are all moved from null to true.